grafana/oss_org_id.go

```diff
@@ -0,0 +1,43 @@
+package grafana
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func makeOSSOrgID(orgID int64, resourceID interface{}) string {
+	return fmt.Sprintf("%d:%s", orgID, fmt.Sprint(resourceID))
+}
+
+func splitOSSOrgID(id string) (int64, string) {
+	if strings.ContainsRune(id, ':') {
+		parts := strings.SplitN(id, ":", 2)
+		orgID, _ := strconv.ParseInt(parts[0], 10, 64)
+		return orgID, parts[1]
+	}
+
+	return 0, id
+}
+
+// nolint: unparam
+func clientFromOSSOrgID(meta interface{}, id string) (*gapi.Client, int64, string) {
+	orgID, restOfID := splitOSSOrgID(id)
+	client := meta.(*client).gapi
+	if orgID > 0 {
+		client = client.WithOrgID(orgID)
+	}
+	return client, orgID, restOfID
+}
```

Comment:
In the two places this function is invoked we're not using the second returned param (`orgID`), does still make sense to return it?

Reply:
I have cases that will use it in the future (this is one of multiple PRs)

Reply:
Cool, looking forward to reviewing those!